plugin/pkg/scheduler/algorithm/well_known_labels.go

```diff
@@ -30,7 +30,7 @@ const (
 	// TaintNodeUnreachable would be automatically added by node controller
 	// when node becomes unreachable (corresponding to NodeReady status ConditionUnknown)
 	// and removed when node becomes reachable (NodeReady status ConditionTrue).
-	TaintNodeUnreachable = "node.alpha.kubernetes.io/unreachable"
+	TaintNodeUnreachable = "node.kubernetes.io/unreachable"
```

Comment:
Why not do this similar with `not-ready`? Add an new label, named `DeprecateTaintNodeUnreachable`.

Reply:
@k82cn Not very sure about this actually,  as `unreachable` seems to be internal used only. @liggitt Do you think we need to address compatibility issue here, just like we did for `not-ready` taint?

Reply:
@liggitt , any suggestion :).

Reply:
similar issue to the other taint name change... the node controller currently adds this taint, so I'd expect it to remove the alpha taint

also, the DefaultTolerationSeconds admission plugin currently adds tolerations for the alpha taint (and isn't [documented](https://kubernetes.io/docs/admin/admission-controllers/#defaulttolerationseconds) to be alpha or subject to change). What's the plan for avoiding disrupting pods admitted with tolerations by that plugin?

Reply:
The approach in #51266 should be able to fix the first concern, i.e. auto-replace old taint to new taint by controller.

Also,  we can do the same operation for `DefaultTolerationSeconds` , let the admission plugin replace old toleration with new one. @liggitt WDYT?

Reply:
> let the admission plugin replace old toleration with new one. @liggitt WDYT?

that doesn't update pods already in the system

Reply:
So add a `admitHandler` to fix pods tolerations on Node's side?

Reply:
So maybe add a `admitHandler` to fix pods tolerations on Node's side? @liggitt

Reply:
I'm not sure the best course of action... @davidopp how did you envision updating system-added tolerations on existing pods when the system-added taint changes names.

Reply:
kindly ping @davidopp , we are discussing the strategy to upgrade alpha taint keys without breaking existing user, and your advice would be highly valued here.

Reply:
Ok, so I will implement the first part 1 first, replace the alpha taint atm in controller.